src/main/java/codeu/model/data/Setting.java

```diff
@@ -0,0 +1,16 @@
+package codeu.model.data;
+
+public class Setting {
+    private String defaultTheme;
+
+    public Setting(String  defaultTheme){
```

Comment:
Lets change this constructor to be more general.

Something like public `public Setting(SettingType type, String value)`

SettingType could look something like this (an enum that takes a parameter):
```
public enum SettingType {
    COLOR ("color"),
    FONT_SIZE("font_size");
    
    private storageKey;
    SettingType (String storageKey) {
      this.storageKey = storageKey;
   }
   public String getStorageKey() {
      return storageKey;
   }
}
```

We could also have other useful methods here like:
`public String getValue()` I think at the start these will return the same thing, but thinking about different setting we have we may want more constructors or getters based on the type a setting is. For instance FONT_SIZE should probably be an int not a String so we'd need a different `getValue()` that returned an int (this point may be a bit confusing ik).
`public String getStorageValue()` this method will convert the value of w.e setting we are looking at to string for storage (since persistent data can only take strings).
`public void setValue(String value)` 
`public SettingType getType()`

Then an example usage of this class is like:
`Setting setting = new Setting(SettingType.COLOR, "red");`
`settingEntity.setProperty(setting.getType().getStorageKey(), setting.getStorageValue());`
etc, etc